scripts/ecs-anywhere-install.sh

```diff
@@ -0,0 +1,526 @@
+    fail
+fi
+
+if [ -f "/sys/fs/cgroup/cgroup.controllers" ]; then
+    echo "Your system is using cgroups v2, which is not supported by ECS."
+    echo "Please change your system to cgroups v1 and reboot. If your system has grubby, we suggest using the following command:"
+    echo '    sudo grubby --update-kernel=ALL --args="systemd.unified_cgroup_hierarchy=0" && sudo shutdown -r now'
+    fail
+fi
+
+# required:
+REGION=""
+ACTIVATION_CODE=""
+ACTIVATION_ID=""
+# optional:
+SKIP_REGISTRATION=false
+ECS_CLUSTER=""
+DOCKER_SOURCE=""
+ECS_VERSION=""
+DEB_URL=""
+RPM_URL=""
+ECS_ENDPOINT=""
+# Whether to check sha for the downloaded ecs-init package. true unless --rpm-url or --deb-url is specified.
+CHECK_SHA=true
+while :; do
+    case "$1" in
+    --region)
+        REGION="$2"
+        shift 2
+        ;;
+    --cluster)
+        ECS_CLUSTER="$2"
+        shift 2
+        ;;
+    --activation-code)
+        ACTIVATION_CODE="$2"
+        shift 2
+        ;;
+    --activation-id)
+        ACTIVATION_ID="$2"
+        shift 2
+        ;;
+    --docker-install-source)
+        DOCKER_SOURCE="$2"
+        shift 2
+        ;;
+    --ecs-version)
+        ECS_VERSION="$2"
+        shift 2
+        ;;
+    --deb-url)
+        DEB_URL="$2"
+        CHECK_SHA=false
+        shift 2
+        ;;
+    --rpm-url)
+        RPM_URL="$2"
+        CHECK_SHA=false
+        shift 2
+        ;;
+    --ecs-endpoint)
+        ECS_ENDPOINT="$2"
+        shift 2
+        ;;
+    --skip-registration)
+        SKIP_REGISTRATION=true
+        shift 1
+        ;;
+    *)
+        [ -z "$1" ] && break
+        echo "invalid option: [$1]"
+        fail
+        ;;
+    esac
+done
+
+if [ -z "$REGION" ]; then
+    echo "--region is required"
+    fail
+fi
+# If activation code is absent and skip activation flag is present, set flag to skip ssm registration
+# if both activation code is present
+if $SKIP_REGISTRATION; then
+    echo "Skipping registering as --skip-registration flag is specified."
+else
+    if [[ -z $ACTIVATION_ID || -z $ACTIVATION_CODE ]]; then
+        echo "Both --activation-id and --activation-code are required unless --skip-registration is specified."
+        fail
+    fi
+fi
+if [ -z "$ECS_CLUSTER" ]; then
+    ECS_CLUSTER="default"
+fi
+if [ -z "$DOCKER_SOURCE" ]; then
+    DOCKER_SOURCE="all"
+fi
+if [ -z "$ECS_VERSION" ]; then
+    ECS_VERSION="latest"
+fi
+
+ARCH=$(uname -m)
+if [ "$ARCH" == "x86_64" ]; then
+    ARCH_ALT="amd64"
+elif [ "$ARCH" == "aarch64" ]; then
+    ARCH_ALT="arm64"
+else
+    echo "Unsupported architecture: $ARCH"
+    fail
+fi
+
+S3_BUCKET="amazon-ecs-agent-packages-preview"
+RPM_PKG_NAME="amazon-ecs-init-$ECS_VERSION.$ARCH.rpm"
+DEB_PKG_NAME="amazon-ecs-init-$ECS_VERSION.$ARCH_ALT.deb"
+
+# TODO [Update before release] Change urls to be region specific buckets
+if [ -z "$RPM_URL" ]; then
+    RPM_URL="https://ecs-init-packages-testing.s3.amazonaws.com/amazon-ecs-init-latest.$ARCH_ALT.rpm"
```

Comment:
this should change to "https://$S3_BUCKET.s3.amazonaws.com/$RPM_PKG_NAME" (and change S3_BUCKET to ecs-init-packages-testing). this way we just need to change the bucket at the end and it allows us to test `--ecs-version` option. same for the deb url

Reply:
Sorry missed that.. Updated.